SCloudRandomizer/MusicSource.h

```diff
@@ -17,7 +17,7 @@ typedef void(^tracksFetchedCompletionHandler)(NSArray* tracks);
 typedef void(^singleTrackFetchedCompletionHandler)(Track* track);
 
 + (MusicSource *) getInstance;
-- (BOOL)isUserLoggedIn;
+- (SCAccount *)isUserLoggedIn;
```

Comment:
Oops this is my bad too, can you update the code to check if SCAccount* is not null and then return a BOOL